solutions/3.js

```diff
@@ -0,0 +1,9 @@
+const solution = (num, hello = '', i = 0) => {
+  if(i == num){
+    return hello;
+  }
+  hello = hello + 'hello';
+  return solution(num, hello, i + 1);
```

Comment:
avoid creating unnecessary variables if you can!
`return solution(num, hello+'hello', i+1);`

Reply:
Okay, just made changes according to this!